app/controllers/derby_events_controller.rb

```diff
@@ -7,14 +7,14 @@ def index
     if admin_signed_in?
       @derby_events = DerbyEvent.all
     else
-      @derby_events = DerbyEvent.where(approved: true, deleted: false).where("start_date >= ? OR end_date >= ?", Date.today, Date.today)
+      @derby_events = DerbyEvent.viewable.where("start_date >= ? OR end_date >= ?", Date.today, Date.today)
```

Comment:
I love scopes so I added one here, I'd probably add another one for the date filter but I'm not really sure what to call it.